languages/csharp/exercises/concept/properties/.meta/Example.cs

```diff
@@ -0,0 +1,59 @@
+using System;
+
+public enum Units
+{
+    Kilograms
+    ,Pounds
+}
+public class WeighingMachine
+{
+    private const float POUNDS_PER_KILOGRAM = 2.20462f;
+    private float inputWeight;
+
+    public Units Units { get; set; } = Units.Kilograms;
+    public float InputWeight
```

Comment:
I like this property. It makes sense as a getter/setter that requires input validation.